template-fr.html

```diff
@@ -104,15 +104,8 @@ <h2 class="wb-inv">Page details</h2><div class="row"><div class="col-sm-8 col-md
 	<div class="landscape">
 		<nav class="container wb-navcurr">
```

Comment:
Remove entire `nav` element. Temporarily add a `style="height: 10em;"` on the `div.landscape` element until we have a response from TBS.